README.md

```diff
@@ -50,7 +55,12 @@ To configure the module behaviour, provide a non-empty JSON object to the `--exe
             // - nimbus: statusim/nimbus-eth2:amd64-latest",
             // - prysm: prysmaticlabs/prysm-beacon-chain:latest,prysmaticlabs/prysm-validator:latest",
             // - lodestar: chainsafe/lodestar:next",
-            "clImage": ""
+            "clImage": "",
+
+            // The log level that the CL client should log at, it overrides the value set in the `logLevel` property
+            // This value must be specific for the client's implementation, For instance if you want to set info log 
+            // level for Teky client, you should set "INFO" and "info" for Lodestar clients
+            "clLogLevel": "",
         }
     ],
 
```

Comment:
I think there's something here that doesn' tmatch the code - the `logLevel` (global) default is being set to `""`, but here it's being left as `info`